nodejs/week1/homework/test/routes.spec.js

```diff
@@ -0,0 +1,85 @@
+const request = require("supertest");
+const app = require("../src/backend/app");
+const meals = require("../src/backend/data/meals");
+const reviews = require("../src/backend/data/reviews");
+const reservations = require("../src/backend/data/reservations");
+
+describe("GET /", () => {
+  test("responds with corrected response", async () => {
+    const response = await request(app).get("/");
+    expect(response.statusCode).toBe(200);
+    expect(response.text).toBe("Meal Sharing Web App");
+  });
+});
+
+describe("GET /meals", () => {
+  test("responds with all meals (with their reviews, if any) from data file", async () => {
+    // Deep copy meals
+    const copiedMeals = JSON.parse(JSON.stringify(meals));
```

Comment:
There is no need to convert meals into a string and then convert it back into an array of objects 🙂